sdk/python/tests/integration/registration/test_inference.py

```diff
@@ -137,6 +139,16 @@ def test_update_data_sources_with_inferred_event_timestamp_col(simple_dataset_1)
                 [file_source], RepoConfig(provider="local", project="test")
             )
 
+    (_, _, data_sources) = universal_data_sources
```

Comment:
I looked through how universal_data_sources is created and the source type is defined by the environment -- https://github.com/feast-dev/feast/blob/master/sdk/python/tests/integration/feature_repos/repo_configuration.py#L416 but im not sure where/how to set that to redshift.

Reply:
you shouldn't need to set to redshift. This will check across bigquery + snowflake + redshift as well.

Reply:
could you split this into a separate test btw? as is it'll actually run the test before this for each repo configuration.

would also mark it as @pytest.mark.universal so it's run when we use make test-python-universal